jekyll/_cci2/ip-ranges.md

```diff
@@ -0,0 +1,146 @@
+- 54.81.162.133
+- 54.83.41.200
+- 54.92.235.88
+
+IP address ranges for core services (used to trigger jobs, exchange information about users between CircleCI and Github etc):
+
+- 18.214.70.5
+- 52.20.166.242
+- 35.174.249.131
+- 18.214.156.84
+- 54.236.156.101
+- 3.210.128.175
+
+**Upcoming Changes to List of IP Address Ranges:
+*Last Updated*: 07/21/2021
+None
+
+**Note:** _jobs can use any of the address ranges above.  It’s also important to note that the address ranges are shared by all CircleCI customers who have opted into using the feature._
+{: class="alert alert-warning"}
+
+Machine-consumable lists can be found below:
+
+- IP address ranges *for jobs*: [DNS A record](https://dnsjson.com/jobs.knownips.circleci.com/A.json).
+
+- IP address ranges *for core services*: [DNS A record](https://dnsjson.com/core.knownips.circleci.com/A.json).
+
+- *All IP address ranges*:  [DNS A record](https://dnsjson.com/all.knownips.circleci.com/A.json).
+
+During the preview phase, this list may change. You should check regularly for updates, at least once a week.  
+ 
+Notifications of a change to this list will be sent out via email to all customers who have at least one job opted into the IP ranges feature. When the feature is generally available, **30 days notice** will be given before changes are made to the existing set of IP address ranges. This page and the machine-consumable list will also be updated when there are upcoming changes.
+
+## Pricing
+{: pricing }
+
+Pricing will be calculated based on network data usage of jobs opted into the IP ranges feature; however, only the traffic of the opted-in jobs will be counted. It is possible to mix jobs with and without the IP ranges feature within the same workflow or pipeline.
+
+Specific rates and details are being finalized and will be published when the feature is generally available. 
+
+## AWS and GCP IP Addresses
+{: awsandgcpipaddresses }
+
+The machines that execute *all jobs* on CircleCI’s platform, not just jobs opted into IP ranges, are hosted on Amazon Web Services (AWS) and Google Cloud Platform (GCP).  An exhaustive list of IP addresses that CircleCI’s traffic may come from on these cloud providers’ platforms can be found by looking up each cloud providers’ IP address ranges. Each cloud provider offers endpoints to find this information.
+ 
+- [AWS](https://ip-ranges.amazonaws.com/ip-ranges.json): CircleCI uses the *us-east-1* and *us-east-2* regions
+- [GCP](https://www.gstatic.com/ipranges/cloud.json): CircleCI uses the *us-east1* and *us-central1* regions
+ 
+CircleCI *does not recommend* configuring an IP-based firewall based on the above IP addresses, as the vast majority are not CircleCI’s machines. Additionally, there is *no guarantee* that the addresses in the endpoints above persist from day-to-day, as these addresses are reassigned continuously.  
+ 
+**IP ranges** is the recommended method for configuring an IP-based firewall to allow traffic from CircleCI’s platform.
```

Comment:
This seems super important and could also be in the intro or use cases list? Just a thought.

Reply:
i thought about that, but i'd rather not call attention to this approach unless someone has gone thru the effort of searching down the page for it.  so thats why i left it out of the intro